types/index.d.ts

```diff
@@ -352,8 +352,34 @@ declare module 'found' {
         }) => React.ReactNode);
   }
 
-  class Link extends React.Component<LinkProps> {
+  type PropsOf<
+    Tag extends React.ReactType
+  > = Tag extends keyof JSX.IntrinsicElements
+    ? JSX.IntrinsicElements[Tag]
+    : Tag extends React.SFC<infer Props>
+    ? Props & React.Attributes
+    : Tag extends React.ComponentClass<infer Props2>
+    ? (Tag extends new (...args: any[]) => infer Instance
+        ? Props2 & React.ClassAttributes<Instance>
+        : never)
+    : never;
+
+  type ReplaceProps<Inner extends React.ReactType, P> = Omit<
```

Comment:
`React.ReactType` is deprecated... see the equivalent PR on 4c/layout for non-deprecated usages

Reply:
I couldn't find a deprecation doc anywhere

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a9ba077376adae5812a96ccdc38a61cba3a36caa/types/react/index.d.ts#L70-L72

does your IDE not show this?

Reply:
uh, yeah, not off the bat (need to click)

Reply:
ping